book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -266,9 +511,9 @@ which should:
 
 1. Obtain the highest entry of `hash_by_height` as `(old_height, old_tip)`.
 Check that `block`'s parent hash is `old_tip` and its height is
-`old_height+1`, or panic.  This check is performed as defense-in-depth
-to prevent database corruption, but it is the caller's responsibility to
-commit finalized blocks in order.
+`old_height+1`, or panic. This check is performed as defense-in-depth to
+prevent database corruption, but it is the caller's responsibility (e.g. the
+zebra-state service's responsibility) to commit finalized blocks in order.
```

Comment:
This is really good to clarify, since we realized we'd need to relax the constraint on the service calls, but I think it would be clearer if we made it explicit who the caller is (the `SledState`).